detekt/wrapper/src/main/java/io/buildfoundation/bazel/detekt/WriterFactory.java

```diff
@@ -0,0 +1,11 @@
+package io.buildfoundation.bazel.detekt;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterFactory {
```

Comment:
You def don't need custom class for this, I've commented above on how to write a string to a file in more Java-canonical way :)

Reply:
Thanks! I forgot to get rid of this after removing mockito.